src/komodo_bitcoind.cpp

```diff
@@ -2489,6 +2497,11 @@ int32_t komodo_staked(CMutableTransaction &txNew,uint32_t nBits,uint32_t *blockt
     memset(utxotxidp,0,sizeof(*utxotxidp));
     memset(utxovoutp,0,sizeof(*utxovoutp));
     memset(utxosig,0,72);
+    CBlockIndex *tipindex = nullptr;
+    {
+        LOCK(cs_main);
+        tipindex = chainActive.Tip();
+    }
     if ( (tipindex= chainActive.Tip()) == 0 )
```

Comment:
seems here it should be just 
`if (tipindex == nullptr)`?

Reply:
Oops. Fixed.